azerothcore/azerothcore-wotlk
# fix(DB/SAI): Bloodmyst Isle missing SAI

Started checking lower level zones for missing SAI or other problems.

Using Kiera DB to make new, so mods, feel free to edit SQL if need.

To test:
.tele Bloodmystisle -> Check creatures around.

Apply SQL and then restart server (Or reload creature template for every ID)
